core/src/main/java/cz/xtf/core/openshift/OpenShift.java

```diff
@@ -93,6 +93,7 @@
     private static volatile String routeSuffix;
 
     public static final String KEEP_LABEL = "xtf.cz/keep";
+    public static final String XTF_GENERATED_LABEL = "xtf.cz/generatedNamespace";
```

Comment:
I'm a bit thinking about this one thing now... I wonder whether the introduction of a completely new label is okay for you or whether usage of already existing `KEEP_LABEL` would be better. I have to say, that I really don't have exact idea what this existing label is supposed to be used for. As such, I'm not sure whether it is suitable also for my usage - labeling namespaces just for the purpose to identify they had been created by XTF.

Reply:
`xtf.cz/keep` has a different purpose, it prevents resources from being cleaned by XTF as part of `OpenShift.clean()` method. The new label is ok for me, we might also generalize it a bit so it can be used also for other resources, `xtf.cz/managed` or something like that.

Reply:
I like that and I agree - although, with this we should put it on all resources so it really does based on its name... Are you okay if I modify it and add a comment there that at the moment we use this label only for the `namespaces` and that it can be extended later?

Reply:
+1

Reply:
Updated in latest `wip` commit